<launch>
    <node name="joy_motor_control_xbox" pkg="pwm_motor_control_ros" type="joy_motor_control_node">
        <param name="axis_motor_forward" value="5" />
        <param name="axis_max_motor_forward" value="-1.0" />
        <param name="axis_min_motor_forward" value="1.0" />

        <param name="axis_motor_backward" value="2" />
        <param name="axis_max_motor_backward" value="-1.0" />
        <param name="axis_min_motor_backward" value="1.0" />
        
        <param name="axis_servo" value="1" />
    </node>
</launch>